webapi/src/test/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala

```diff
@@ -129,9 +129,10 @@ class OntologyV2R2RSpec extends R2RSpec {
      */
     def writeFile(responseStr: String, mediaType: MediaType.NonBinary): Unit =
       if (!disableWrite) {
-        val testOutputDir = Paths.get("..", "test_data", "ontologyR2RV2")
-        val file = makeFile(mediaType)
```

Comment:
Is there a reason why not to use makeFile anymore? Are there other cases where it can be replaced (and the method removed if it is not used anymore)?

Reply:
Because it simply wasn't work as expected.